cmd/bootstrap/utils/key_generation_test.go

```diff
@@ -62,7 +64,7 @@ func TestWriteMachineAccountFiles(t *testing.T) {
 	expected := make(map[string]bootstrap.NodeMachineAccountInfo)
 	for i, node := range nodes {
 		// See comments in WriteMachineAccountFiles for why addresses take this form
-		addr, err := chain.AddressAtIndex(uint64(6 + i*2))
+		addr, err := chain.AddressAtIndex(uint64(systemcontracts.LastSystemAccountIndex + (i+1)*2))
```

Comment:
@jordanschalm can you confirm this logic is correct?

Reply:
It looks correct to me 👍